src/Presentation/Nop.Web.Framework/Mvc/Filters/HttpsRequirementAttribute.cs

```diff
@@ -119,6 +119,10 @@ public void OnAuthorization(AuthorizationFilterContext filterContext)
                 if (!DataSettingsManager.DatabaseIsInstalled)
                     return;
 
+                //ignore this rule for localhost
+                if (_webHelper.IsLocalRequest(filterContext.HttpContext.Request))
+                    return;
```

Comment:
This code is not related to solving the problem. Please exclude it from the pull request.

Reply:
@DmitriyKulagin its done, thanks for reviewing it.